### PR TITLE
feature: add multiple placeholders

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: cluster/manifests/skipper/deployment.yaml [hunk 1]
```diff
@@ -206,7 +206,9 @@ spec:
           - '-default-filters-prepend={{ .ConfigItems.skipper_default_filters }}'
           - "-disabled-filters={{ .ConfigItems.skipper_disabled_filters }}"
 {{ if ne .ConfigItems.skipper_edit_route_placeholders "" }}
-          - '-edit-route={{ .ConfigItems.skipper_edit_route_placeholders }}'
+          {{ range $pattern := split .ConfigItems.skipper_edit_route_placeholders "[cf724afc]" }}
```
Comment (AlexanderYastrebov) [inline cluster/manifests/skipper/deployment.yaml]: Maybe rename $pattern to $placeholder ?

Comment (szuecs) [reply cluster/manifests/skipper/deployment.yaml]: fixed

**Comment by szuecs** [comment]
failing tests seem to be something else, not completely sure but pod was terminated in e2e test

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
validated change in dev cluster

**Comment by szuecs** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**